objects/src/accounts/account_id.rs

```diff
@@ -289,3 +289,40 @@ pub fn compute_digest(seed: Word, code_root: Digest, storage_root: Digest) -> Di
 pub fn digest_pow(digest: Digest) -> u32 {
     digest.as_elements()[3].as_int().trailing_zeros()
 }
+
+#[cfg(test)]
+mod test {
+    use super::{AccountId, AccountType};
+
+    const ACCOUNT_ID_REGULAR_ACCOUNT_IMMUTABLE_CODE_ON_CHAIN: u64 = 0b0110011011u64 << 54;
+    const ACCOUNT_ID_REGULAR_ACCOUNT_UPDATABLE_CODE_OFF_CHAIN: u64 = 0b0001101110 << 54;
+    const ACCOUNT_ID_FUNGIBLE_FAUCET_ON_CHAIN: u64 = 0b1010011100 << 54;
+    const ACCOUNT_ID_NON_FUNGIBLE_FAUCET_OFF_CHAIN: u64 = 0b1101100110 << 54;
```

Comment:
These are good examples of what I meant on https://github.com/0xPolygonMiden/miden-base/issues/250#issuecomment-1728209285

They serve the same purpose as:

https://github.com/0xPolygonMiden/miden-base/blob/f0e27957cbaddad64ee132eb9e6896517a28d65f/mock/src/constants.rs#L16-L18

But we have to duplicate it here to avoid the cyclic dependency.

Reply:
Could we not export these constants from `miden-base` under `testing` feature, and then import them in `mock` instead of redefining them there?